component/site/views/default/helpers/defaultloadedfromtemplate.php

```diff
@@ -1102,6 +1103,26 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 					    $replace[] = $fieldval;
 					    $blank[] = "";
 
+
+					    $search[] = "{{PAST_OR_FUTURE}}";
+					    $timedelta = $row->getUnixStartTime() - JevDate::mktime();
+					    $eventPassed = !($timedelta >= 0);
+					    $shownsign = false;
+					    $fieldval = 'past';
+					    if (stripos($fieldval, "%nopast") !== false)
```

Comment:
the code should read
`					    $search[] = "{{PAST_OR_FUTURE}}";
					    $timedelta = $row->getUnixStartTime() - JevDate::mktime();
					    $eventPassed = !($timedelta >= 0);
					    $shownsign = false;
					    if (!$eventPassed)
					    {
						    $replace[] =  'future';
					    }
					    else
					    {
						    $replace[] =  'past';
					    }
					    $blank[] = "";
`